odoo_conn.php

```diff
@@ -1,36 +1,38 @@
 <?php
-/*
-Plugin Name: Contact 7 to Odoo connector
-Plugin URI: https://www.jackdane.co.uk
-Description: Connect your WordPress Contact 7 Forms to Odoo
-Version: 0.1.2
-Requires PHP: 7.3
-Author: Jack Dane
-Author URI: https://www.jackdane.co.uk
-*/
-
-add_option("odoo_conn_db_version");
-
-require("vendor/autoload.php");
-
-require("encryption.php");
-
-require("odoo_connector/odoo_connector.php");
-
-require("dependency_check.php");
-
-register_activation_hook(__FILE__, "\\odoo_conn\\dependency_check\\odoo_conn_contact_7_plugin_active");
-
-require("activation.php");
-
-register_activation_hook(__FILE__, "odoo_conn_activation_function");
-
-require("deactivation.php");
-
-register_deactivation_hook(__FILE__, "odoo_conn_deactivation_function");
-
-require("loaded.php");
-
-require("admin/main.php");
-
-?>
\ No newline at end of file
+/**
+ * Plugin Name: Contact Form 7 to Odoo connector
+ * Description: When submitting a form through Contact Form 7, the data is extracted and mapped to the designated fields in Odoo.
+ * Author: Jack Dane
+ * Author URI: https://www.jackdane.co.uk
+ * Plugin URI: https://github.com/Jack-Dane/odoo-wp-plugin
+ * Requires PHP: 7.3
+ * Tested up to: 6.5.3
+ * Text Domain: cf7-odoo-sync
+ * Domain Path: /languages
```

Comment:
I don't think this is correct if I understand the documentation correctly, this plugin doesn't have any translations. 
> The domain path defines the location for a plugin’s translation. This has a few uses, notably so that WordPress knows where to find the translation even when the plugin is disabled. This defaults to the folder in which your plugin is found.
   For example, if the translation is located in a folder called languages within your plugin, then the Domain Path is /languages and must be written with the first slash:
https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#domain-path

Reply:
I would like to translate your plugin into Spanish if you think it is appropriate.

Reply:
Yep, sounds great, this Domain Path can stay then :+1: